models/bookDao.js

```diff
@@ -172,9 +172,36 @@ const isExistedBook = async (bookId) => {
   }
 };
 
+const createReview = async (userId, bookId, content, score) => {
+  console.log(userId, bookId, content, score);
+  try {
+    const result = await dataSource.query(
+      `INSERT INTO reviews (
+        user_id,
+        book_id,
+        content,
+        score
+        )
+      VALUES(
+        ?,
+        ?,
+        ?,
+        ?
+        )`,
+      [userId, bookId, content, score]
+    );
+    return result;
+  } catch (err) {
+    const error = new Error('INVALID_DATA_INPUT');
+    error.statusCode = 500;
```

Comment:
그 동안 400으로 잘 쓰다가 왜 갑자기 다시 500이 나왔죠?
```suggestion
    error.statusCode = 400;
```